Kitodo/src/main/java/org/kitodo/production/helper/tasks/UpdateInternalMetaInformationTask.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.production.helper.tasks;
+
+import java.util.List;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.kitodo.data.database.beans.Process;
+import org.kitodo.data.database.beans.Project;
+import org.kitodo.data.exceptions.DataException;
+import org.kitodo.production.helper.Helper;
+import org.kitodo.production.services.ServiceManager;
+import org.kitodo.production.services.data.ProcessService;
+
+
+public class UpdateInternalMetaInformationTask extends EmptyTask {
+
+    private static final Logger logger = LogManager.getLogger(UpdateInternalMetaInformationTask.class);
+    private final Project project;
+    private final ProcessService processService;
+
+    /**
+     * Default constructor.
+     *
+     * @param project Chosen projet which process meta information should be updated.
```

Comment:
```suggestion
     * @param project Chosen project which process meta information should be updated.
```

Reply:
Same typo exists in main branch pull request. Should I fixed it in the other pull request too?